src/lib/hooks/liquidity-hub/useLiquidityHub.ts

```diff
@@ -0,0 +1,12 @@
+import { useMemo } from "react";
+import { constructSDK } from "@orbs-network/liquidity-hub-sdk";
+import { useAccount } from "wagmi";
+
+export const useLiquidityHub = () => {
+  const { chainId } = useAccount();
+
+  return useMemo(
+    () => constructSDK({ chainId: chainId, partner: "widget" }),
```

Comment:
i like this but I think we can use a better name like `initialiseLiquidityHub` or even turn it into a class like:

```
() => new LiquidityHubSDK({ chainId: chainId, partner: "widget" }),
```

or something like this